{
	"id": "00000000-0000-0000-dc2e-0000ad57a984",
	"architectureId": 0,
	"baseProductName": "SQL Server 2019 Big Data Clusters",
	"baseProductVersion": "0",
	"cpe": "cpe:2.3:o:microsoft:sql_server_big_data_clusters:2019:*:*:*:*:*:*:*",
	"customerActionRequired": true,
	"cveNumber": "CVE-2021-44228",
	"cweDetailsList": [],
	"cweList": [],
	"impact": "Remote Code Execution",
	"impactId": 100000005,
	"initialReleaseDate": "2021-12-16T08:00:00Z",
	"isMariner": false,
	"issuingCna": "Apache Software Foundation",
	"kbArticles": [
		{
			"affectedBinaries": [],
			"articleName": "Additional Information",
			"articleUrl": "https://msrc-blog.microsoft.com/2021/12/11/microsofts-response-to-cve-2021-44228-apache-log4j2/#SQL-Server-2019-Big-Data-Clusters",
			"downloadName": "Security Update",
			"ordinal": 0,
			"rebootRequired": "Maybe"
		}
	],
	"platformId": 0,
	"product": "SQL Server 2019 Big Data Clusters",
	"productFamily": "SQL Server",
	"productFamilyId": 100000008,
	"productId": 11996,
	"releaseDate": "2021-12-16T08:00:00Z",
	"releaseNumber": "2021-Dec",
	"severity": "Critical",
	"severityId": 100000000
}
